app/views/dashboards/_actions.html.erb

```diff
@@ -1,70 +1,93 @@
 <nav class="hidden m:block" aria-label="Dashboards">
-  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "show" && (params[:which] == "organization" || params[:which].blank?) %>"
-    href="<%= dashboard_path %>"
-    <%= params[:action] == "show" && (params[:which] == "organization" || params[:which].blank?) ? ' aria-current="page"'.html_safe : "" %>>
-    Posts
-    <span class="crayons-indicator"><%= @user.articles_count %></span>
-  </a>
+  <ul>
```

Comment:
One thing I didn't explicitly mention on the issue is that visually we don't wany any change as a result of adding the list. We use tailwind-style classes, and I _think_ you can achieve this with the suggestion below (but probably best to double check!)

```suggestion
  <ul class="list-none p-0">
```

Reply:
I tested @aitchiss's suggestion and it does work! 👍🏾